pentaho-requirejs-osgi-manager/src/main/java/org/pentaho/js/require/RequireJsConfigManager.java

```diff
@@ -168,7 +167,7 @@ private boolean parsePackageInformation( Bundle bundle, URL resourceUrl ) {
         return true;
       }
     } catch ( Exception ignored ) {
-      // ignored
+      ignored.printStackTrace();
```

Comment:
We don't do this. Please replace with a proper log statement if you feel it necessary